api_app/analyzers_manager/general.py

```diff
@@ -45,16 +45,25 @@ def start_analyzers(
             module = ac.get("python_module", None)
             if not module:
                 raise AnalyzerConfigurationException(
-                    f"no python_module available in config for {analyzer} analyzer?!"
+                    f"no python_module available in config for {analyzer} analyzer?!"  # noqa: E501
                 )
 
-            additional_config_params = ac.get("additional_config_params", {})
+            # Get analyzer config and secrets
+            config_params = ac.get("config", {})
+            secret_params = ac.get("secrets", {})
+            additional_config_params = {}
+            for name, conf in secret_params.items():
+                additional_config_params[name] = conf["value"]
```

Comment:
what is `conf["value"]` ?

Reply:
```json
"secrets": {
            "api_key_name": {
                "value": "MWDB_KEY",
                "type": "string",
                "required": true,
                "default": null,
                "description": "",
                }
           }
```
I found it better to name it `value` instead of `secret_name` because `api_key_name` already denotes secret name and `MWDB_KEY` is it's value. Maybe I could change it to `secret_value` for the sake of clarity?

Reply:
This is confusing. `value` especially is confusing because the field does not the _value_ instead only the name of the environment variable. Do u think you could rename it to `env_var_key` ?